bundle/metadata/dependencies.yaml

```diff
@@ -3,3 +3,7 @@ dependencies:
     value:
       packageName: csi-addons
       version: 0.8.0
+  - type: olm.package
+    value:
+      packageName: noobaa-operator
```

Comment:
odf/ocs-operator also brings the noobaa operator. When odf is deployed in provider mode, I think we will have multiple noobaa operators. Will this cause any problem or we are okay with it?

Reply:
i see this question already being discussed you can resolve this conv.